.github/workflows/performance-benchmarks.yml

```diff
@@ -0,0 +1,35 @@
+name: Performance Benchmarks
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    paths:
+      - 'Sources/**'
+      - 'Samples/TrendingMovies/TrendingMovies-Benchmarking-UITests/**'
+      - '.github/workflows/performance-benchmarks.yml'
+
+jobs:
+  benchmark-with-movies:
+    runs-on: macos-11
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v3
```

Comment:
Didn't know that there is v3 here already. We can update our other actions https://github.com/getsentry/sentry-cocoa/pull/1845